### PR TITLE
Create a-very-useful-data-structure-linked-lists.md

### LINKED ISSUE #1571: Importance of Linked Lists

### PR DESCRIPTION
**Pull request by jainaryan04**
fixes #1571

### CODE REVIEW

**Comment by jainaryan04** [comment]
@ajay-dhangar please merge

File: blog/a-very-useful-data-structure-linked-lists.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    struct Node* head = NULL;
+
+    insertAtBeginning(&head, 3);
+    insertAtBeginning(&head, 2);
+    insertAtBeginning(&head, 1);
+
+    printList(head);
+
+    return 0;
+}
```
Comment (ajay-dhangar) [inline blog/a-very-useful-data-structure-linked-lists.md:97]: close your code example with ```` ``` ````

File: blog/a-very-useful-data-structure-linked-lists.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+---
+slug: a-very-useful-data-structure-linked-lists
+title: A Very Useful Data Structure - Linked Lists
+authors: [Aryan-Jain]
```
Comment (ajay-dhangar) [inline blog/a-very-useful-data-structure-linked-lists.md:4]: write your `Aryan-Jain` name and basic info in the `blog/authors.yml` file like https://github.com/ajay-dhangar/algo/blob/7814b98e8e0993e3b52f573f23810bb19afd453c/blog/authors.yml#L32

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
update your code based on suggestions

**Comment by jainaryan04** [comment]
done @ajay-dhangar

**Review by ajay-dhangar — APPROVED** [review]
Done

**Pull request merged**